packages/components/src/date-time/date.js

```diff
@@ -70,43 +70,40 @@ function DatePickerDay( { day, events = [] } ) {
 	);
 }
 
-class DatePicker extends Component {
-	constructor() {
-		super( ...arguments );
-
-		this.onChangeMoment = this.onChangeMoment.bind( this );
-		this.nodeRef = createRef();
-		this.onMonthPreviewedHandler = this.onMonthPreviewedHandler.bind(
-			this
-		);
-	}
-
-	onMonthPreviewedHandler( newMonthDate ) {
-		this.props.onMonthPreviewed?.( newMonthDate.toISOString() );
-		this.keepFocusInside();
-	}
+function DatePicker( {
+	currentDate,
+	onChange,
+	events,
+	isInvalidDate,
+	onMonthPreviewed,
```

Comment:
A bit of a nit (and one may argue out of scope), but I noticed that the `onMonthPreviewed` prop isn't listed in the [README](https://github.com/WordPress/gutenberg/blob/cb8c98adc9a9071b2f6151499b6cc53d4900df56/packages/components/src/date-time/README.md#props) — would you mind adding it?

Reply:
I have added `onMonthPreviewed` props in the README.md file. Please check it once, I know I may have not written enough info but I tried my best to update the docs and not to confuse users. 😅